tools/mdanalysis/ramachandran_plots.xml

```diff
@@ -45,150 +45,78 @@
         <expand macro="analysis_inputs"/>
         <section name="phi" title="Phi" expanded="False">
           <param name="segid1"  type="text" value="HET" label="Segid of atom 1">
-            <sanitizer invalid_char="">
-              <valid initial="string.ascii_letters,string.digits">
-              </valid>
-            </sanitizer>
+            <expand macro="sanitizer"/>
           </param>
           <param name="resid1"  type="text" value="3" label="Resid of atom 1">
-            <sanitizer invalid_char="">
```

Comment:
that was digits before  now it is letters and digits, is that ok?

Reply:
Fixed it